### PR TITLE
Remove includeDeleted flag from reporting API, should never have been exposed Also add some clarification on filtering on ad statuses

### PR DESCRIPTION
**Pull request by remenska**

### CODE REVIEW

File: src/source/index.rst [hunk 1]
```diff
@@ -48,10 +48,6 @@ History
 
 ..  role:: strike
 
-* v.0.114 - 6 Jan 2020 - tw
```
Comment (twessling-icas) [inline src/source/index.rst:51]: make new version 0.115, saying this is undone and now always like that

File: src/source/reporting.rst [hunk 2]
```diff
@@ -206,6 +206,8 @@ The only valid ``operator`` at the moment is the set operator ``in``, and requir
     ]
 
 
+.. note:: Currently the API **does not allow to filter on the current status of ads**, which means the reporting numbers apply for **all** ads, including those that have been deleted in the meantime. Depending on API clients feedback, we might extend support for such filtering in the future.
```
Comment (twessling-icas) [inline src/source/reporting.rst]: you say "Currently.." but I don't think we have any intention of changing that?

**Review by twessling-icas — CHANGES_REQUESTED** [review]

File: src/source/reporting.rst (hunk 2 above)
Comment (remenska) [reply src/source/reporting.rst]: no indeed, I'll rephrase it a bit. 
Who knows in the distant future we may be able to pull it off if there is a demand for it

File: src/source/index.rst (hunk 1 above)
Comment (remenska) [reply src/source/index.rst:51]: done & done

**Review by twessling-icas — APPROVED** [review]

**Pull request merged**